api/src/main/resources/workbench.yaml

```diff
@@ -663,6 +663,52 @@ paths:
           schema:
             $ref: "#/definitions/CriteriaListResponse"
 
+  # Cohort Review  #######################################################################
+  /api/v1/cohortreview/participants/{cohortId}/{cdrVersionId}:
+    get:
+      tags:
+         - cohortReview
+      description:  Returns a collection of participants that for the specified cohortId and cdrVersionId
+      operationId: "getParticipants"
+      parameters:
+        - in: path
+          name: cohortId
+          type: integer
+          format: int64
+          required: true
+          description: specifies which cohort
+        - in: path
+          name: cdrVersionId
+          type: integer
+          format: int64
+          required: true
+          description: specifies version of cdr
+        - in: query
+          name: page
+          type: integer
+          required: false
+          description: specific page (default is 0)
+        - in: query
+          name: limit
+          type: integer
+          required: false
+          description: page size of results(default is 25)
```

Comment:
nit: Missing a space before parents `results(default` => `results (default`

Same nit at lines [700](https://github.com/all-of-us/workbench/pull/302/files#diff-b0dcdb860ed0db8c51e9bfadb75b2d07R700) and [705](https://github.com/all-of-us/workbench/pull/302/files#diff-b0dcdb860ed0db8c51e9bfadb75b2d07R705)

Reply:
added spaces.